### PR TITLE
Written first version of README.md

### PR DESCRIPTION
**Pull request by jonahbellemans**
Added first version of a readme file with brief instructions about the usage of the platform.

### CODE REVIEW

**Review by MidasLamb — CHANGES_REQUESTED** [review]
Update the image to the new design. Also fix merge conflict please.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# CSSchedule
+
+## Introduction
+
+This repository contains the code for a site that generates ICS calendar files with the individual schedules of Computer Science students at KU Leuven.
+This schedule can be imported in most calendar apps, such as Google calendar or iCal. English courses are taken into consideration as counterparts for their Dutch equivalents.
```
Comment (MidasLamb) [inline README.md]: The counterparts have to be selected manually still.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Usage
+
+To generate an ICS file, the user simply marks the courses they want to include in the list. Afterwards, the user can generate the corresponding ICS calendar by clicking the "Download ICS File" button.
+
+![Screenshot Usage](/images/Screenshot_20171002_142820.png)
```
Comment (MidasLamb) [inline README.md]: Outdated image

**Review by MidasLamb — CHANGES_REQUESTED** [review]
Fix merge plz.

**Comment by jonahbellemans** [comment]
All reviews should be fixed now.

**Review by MidasLamb — APPROVED** [review]

**Pull request merged**